src/SignhostAPIClient/Rest/JsonStringExtensions.cs

```diff
@@ -0,0 +1,40 @@
+using Newtonsoft.Json;
```

Comment:
Don't we want to use `#nullable enable` here?

Reply:
We're still at C# 7.3 and I don't know if we want to C# 8.0 or greater just jet because it's a client library.